bundles/org.openhab.binding.upnpcontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -143,27 +143,47 @@
 		<label>Loudness</label>
 		<description>Loudness</description>
 		<category>SoundVolume</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>SoundVolume</tag>
+		</tags>
 	</channel-type>
 	<channel-type id="stop">
 		<item-type>Switch</item-type>
 		<label>Stop</label>
 		<description>Stop the player</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>MediaControl</tag>
+		</tags>
 		<autoUpdatePolicy>veto</autoUpdatePolicy>
 	</channel-type>
 	<channel-type id="repeat">
 		<item-type>Switch</item-type>
 		<label>Repeat</label>
 		<description>Repeat the selection</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
+		</tags>
 	</channel-type>
 	<channel-type id="shuffle">
 		<item-type>Switch</item-type>
 		<label>Shuffle</label>
 		<description>Random shuffle the selection</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
+		</tags>
 	</channel-type>
 	<channel-type id="onlyplayone">
 		<item-type>Switch</item-type>
 		<label>Only Play One</label>
 		<description>Stop playback after playing one media entry from queue</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>MediaControl</tag>
```

Comment:
```suggestion
			<tag>Mode</tag>
```

Reply:
I will handle this in https://github.com/openhab/openhab-addons/pull/18544